cpp/src/fil/fil.cu

```diff
@@ -689,11 +689,11 @@ void tl2fil_common(forest_params_t* params, const tl::Model& model,
       ASSERT(tl_params->output_class,
              "output_class==true is required for multi-class models");
       ASSERT(pred_transform == "sigmoid" || pred_transform == "identity" ||
-               pred_transform == "max_index" ||
+               pred_transform == "max_index" || pred_transform == "softmax" ||
```

Comment:
even thought softmax is not implemented, we error on predict_proba() on multiclass models, so this will not cause a quiet error